framework/auth/oauth_scopes.py

```diff
@@ -124,7 +124,7 @@ class ComposedScopes(object):
     USERS_READ = (CoreScopes.USERS_READ, CoreScopes.ALWAYS_PUBLIC, )
     USERS_WRITE = USERS_READ + (CoreScopes.USERS_WRITE,)
     USERS_CREATE = USERS_READ + (CoreScopes.USERS_CREATE, )
-    USERS_EMAIL = (CoreScopes.USERS_EMAIL, )
+    USER_EMAIL_READ =  (CoreScopes.USERS_READ, CoreScopes.USER_EMAIL_READ, )
```

Comment:
Please keep this `(CoreScopes.USERS_EMAIL, )` for the time being.

Documentation should be updated to indicate this is additive and requires `USERS_READ` aka `osf.users.profile_read`.

The reason here is we won't get an explicit bullet point when the user authorizes the scopes on CAS and it should be spelled out you are releasing your email address and profile information.